src/base/tools.lua

```diff
@@ -31,6 +31,7 @@
 
 	function p.tools.normalize(identifier)
 		if identifier:startswith("v") then -- TODO: this should be deprecated?
+			p.warnOnce(identifier, "toolset '%s' is deprecated, toolset 'msc-%s' instead.", identifier, identifier)
```

Comment:
I'm fairly certain we've had this discussion before, and it was decided that we wanted `vXXX` to be accepted forever, so deprecation made absolutely no sense.

Reply:
We did that have discussion; I wanted to provide that as a Visual Studio friendly value. The quorum of core developers is larger now, and I don't mind if I'm overruled.

Reply:
I see the value in keeping this.

What are the arguments to deprecate it?